src/prompts.ts

```diff
@@ -48,13 +47,16 @@ const PROMPT_QUESTIONS: PromptObject[] = [
     initial: 0,
   },
   {
-    type: "confirm",
+    type: "select",
     name: "runInstall",
-    message: () => {
-      const packageManager = getUserPkgManager()
-      return `Would you like to run \`${packageManager} install\` after finishing up?`
-    },
-    initial: true,
+    message: "Would you like to install packages after finishing up, if so choose your package manager?",
```

Comment:
I'd write it like that
```suggestion
    message: "Would you like to install packages after finishing up ? If so, choose your package manager.",
```